### DISCUSSION TITLE
add audio input to the mix

### DISCUSSION THREAD

**Discussion by codekoch** [probonopd/MiniDexed#465] [Q&A]
I am very excited about the minidexed project. 
For the perfect setup it would be great if you could add another audio source to the mix via the audio input..e.g. an external drum machine. Then you would only need a cable to the speaker and no external mixer.
Does anyone have an idea how to realize this?

**Comment by probonopd** [comment]
First of all, you'd need a high quality audio input for the Raspberry Pi. At that point, it's probably easier and less expensive to use an external mixer?
